src/core-recipe.jl

```diff
@@ -137,29 +157,3 @@ function enclosing_geometry(::Type{Rect}, positions, enlarge=0.0)
     mini = minimum(rect) .- ((padded_w .- w) ./ 2)
     return Rect2f(mini, padded_w)
 end
-
-"""
-    pad_boundary(::Type{Geometry}, positions, enlarge=0.2) where Geometry
-
-Adds new points to positions, adding the boundary from enclosing all positions with `Geometry`.
-See [`TopoPlots.enclosing_geometry`](@ref) for more details about the boundary.
-"""
-function pad_boundary!(::Type{Geometry}, positions, enlarge=0.2) where Geometry
-    c = enclosing_geometry(Geometry, positions, enlarge)
-    return append!(positions, decompose(Point2f, c))
-end
-
-function pad_data(data::AbstractVector, positions::AbstractVector, value::Number)
-    pad_data(data, length(positions), value)
-end
-
-function pad_data(data::AbstractVector, npositions::Integer, value::Number)
-    ndata = length(data)
-    if npositions == ndata
-        return data
-    elseif npositions < ndata
-        error("To pad the data for new positions, we need more positions than data points")
-    else
-        vcat(data, fill(value, npositions - ndata))
-    end
-end
```

Comment:
let's add these back in since they're causing the docs build to fail and #12 should actually use them

Reply:
I disagree. `pad_data` and `pad_boundary!` work only by padding the data with `pad_value` along the perimeter, which is what caused issue #9. They are never used in this PR. Instead of adding those back, I removed their links in the documentations, which fixes the problem with building the docs -- locally (see if it works in CI).